### DISCUSSION TITLE
Erreur exécution geoclimate

### DISCUSSION THREAD

**Discussion by LesMerlusFrits** [orbisgis/geoclimate#795] [Q&A] [answered]
Bonjour,

Nous avons un message d'erreur lors de l'exécution du fichier de configuration, que ce soit avec OSM ou la BD TOPO v2.

Message d'erreur pour le workflow de la BD TOPO : 
**"The input datasource must be defined with the name bdtopo_v2
Cannot execute the BDTOPO_V2.2 workflow"**

![Erreur BD TOPO](https://user-images.githubusercontent.com/121180173/208937064-257b38c9-c413-40eb-a2a4-7cec64be127f.png)

Erreur pour OSM :
![Erreur filtre OSM](https://user-images.githubusercontent.com/121180173/208946130-2338f9b1-7831-4f24-9ca3-3d3427e1f23c.png)

Merci par avance pour votre aide.
Les Merlus Frits (IGARUN)

**Comment by GDDS2022** [comment, marked as answer]
Bonjour,

je confirme être dans le même cas de figure, je prends les fichiers du wiki, je suis le tuto et j'ai les mêmes erreurs, que ce soit OSM ou BD TOPO.

MAJ : faut utiliser la dernière version snapshot semblerait-il!
